python/cuml/dask/neighbors/kneighbors_classifier.py

```diff
@@ -46,13 +46,9 @@ class KNeighborsClassifier(NearestNeighbors):
     ----------
     n_neighbors : int (default=5)
         Default number of neighbors to query
-    algorithm : string (default='brute')
-        The query algorithm to use. Currently, only 'brute' is supported.
-    metric : string (default='euclidean').
-        Distance metric to use.
-    weights : string (default='uniform')
-        Sample weights to use. Currently, only the 'uniform' strategy is
-        supported.
+    batch_size: int (optional, default 1024)
+        Maximum number of queries processed at once. This parameter can
+        greatly affect the throughput of the algorithm.
```

Comment:
See suggestion below. It might be a good idea for us to start providing information about how to find good settings for these parameters. Can you also update the `NearestNeighbors` pydoc so they are consistent?